scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Scripts
+## prepare-release.sh
+This script creates a release csv for the operator under the `olm-catalog` using `operator-sdk generate csv`
+
+**Required System Variables**
+
+- SEMVER -> valid x.y.z format. Usage: `SEMVER=2.6.0`
+
+**Optional System Variables**
+
+- TYPE -> Which resource the release csv will be created under. Current Default: `integreatly-operator`.  Usage: `TYPE=managed-api-service`
+- ORG -> The org to where any images maybe stored on quay.io. Setting this will change the affect the location images 
```

Comment:
`the affect the`: does this need to be reworded?

Reply:
Yes, yes it does. Even when I read it not it don't make must sense not.

Reply:
@obrienrobert I have just push just push an update if you would like to review

Reply:
I would reword this section: `The org to where any images maybe stored on quay.io`. Maybe something like `The quay.io org to where the images will be pushed` (or something to that effect).

Reply:
I have done these changes locally, there maybe be other changes need to the script part. But yes `maybe` might not have been the right word to use.